download.md

```diff
@@ -17,17 +17,23 @@ See: Official Pup build recipes at [Woof-CE on GITHUB]
 |XenialPup64     |UbuntuLTS16.04 compatible Official Pup (64bit)    |[Xenialpup64 Forum Page][ux64]|
 |Tahrpup         |UbuntuLTS14.04 compatible Official Pup (32bit)    |[Tahrpup Forum Page][ut32]    |
 |Tahrpup64       |UbuntuLTS14.04 compatible Official Pup (64bit)    |[Tahrpup64 Forum Page][ut64]  |
-|ScPup           |SlackwareCurrent compatible Pup (32 & 64bit)      |[ScPup Forum Page][slc]      |
+|S15Pup          |Slackware15.0 compatible Official Pup (32 & 64bit)|[S15Pup Forum Page][sl15]     |
```

Comment:
download.md is very hard to reach, the link is hidden pretty well. Putting a Puppy family distro in download.md is almost the same as removing it from the homepage.